lnrpc/rpc.proto

```diff
@@ -237,6 +240,21 @@ message NewAddressResponse {
     string address = 1 [ json_name = "address" ];
 }
 
+message SignMessageRequest {
+    bytes msg = 1 [ json_name = "msg" ];
+}
+message SignMessageResponse {
+    string signature = 1 [ json_name = "signature" ];
+}
+
+message VerifyMessageRequest {
+    bytes msg = 1 [ json_name = "msg" ];
+    string signature = 2 [ json_name = "signature" ];
+}
+message VerifyMessageResponse {
+    bool valid = 1 [ json_name = "valid" ];
```

Comment:
In-line with @AndrewSamokhvalov's comment on the original issue, this should also return the compressed public key of the signer that was extracted from the signature. This will allow higher-level to enforce any sort of channel capitalization constraint themselves as additional checks after the signature validates.